Sources/HarmonyTesting/ObjectMother/IntegerObjectModel.swift

```diff
@@ -14,13 +14,8 @@
 // limitations under the License.
 //
 
-#if DEBUG
-
 import Foundation
 
-/* These functions are internal to only be retrieved using '@testable import Harmony'. */
-
-func anyInt(minValue: Int = -32000, maxValue: Int = 32000) -> Int {
+public func anyInt(minValue: Int = -32000, maxValue: Int = 32000) -> Int {
```

Comment:
Should all the test classes be public now? The idea was to have them internal so they can only be used on the test classes with import @testable.

Reply:
Nevermind, it's on Harmony Testing now. :)

Reply:
exactly, we moved it to HarmonyTesting